### ISSUE TITLE
Mac VoiceOver doesn't read Datepicker

### ISSUE THREAD

**Issue by shimy1984** [angular-ui/bootstrap#3095]
When using Mac OS X Mavericks, no matter the browser (Chrome, Safari, Firefox, most recent versions) or Quick Nav setting, VoiceOver does not recognize the Datepicker. ChromeVox works as a work around but this is a pretty major accessibility issue.

**Comment by rvanbaalen** [comment]
@shimy1984 Please keep an eye on #3526. Accessibility will get the attention it deserves soon.

Do you happen to use voice overs / screenreaders a lot or were you just testing it? If you're using it yourself, any feedback & tips on the accessibility progress in #3526 would be highly appreciated. 

Is this still an issue in the current master by the way?

**Comment by shimy1984** [comment]
I've actually done a great deal of work related to this already for the project I'm working on. 

I'll see if I can put together a clean pull within the next couple weeks for you all to review. Just gotta work it into my schedule. 

On a side note, I've added some pretty cool enhancements to the the various components, such as a drop up functionality to the date picker.

I'll create another pull when I get the time for those to see if you dudes dig the changes. :)

**Comment by wesleycho** [comment]
Any luck on a PR @shimy1984? We would love to have accessibility support, especially if you have done a bunch of work on it already.

**Comment by wesleycho** [comment]
I did a bit of investigation, and it seems to not read the datepicker on a static instance, but it reads the popup version. It needs to output information on the date when hovering over a date button.

**Comment by icfantv** [comment]
Closing in favor of #4772.

**Comment by dadoadk** [comment]
I seem to be having this issue with Chromevox screen reader. It happens when in Chrome browser Chromevox screen reader does not read the numbers within the datepicker. While using the arrow keys to select a date the Chromevox screen reader does not announce it. I've posted the issue on Chromevox Issue Tracker https://bugs.chromium.org/p/chromium/issues/detail?id=699725#c2

While this same issue does not happen using IE11 and JAWS screen reader.
